search-commons/src/main/java/no/unit/nva/search/resource/ResourceSearchQuery.java

```diff
@@ -119,15 +120,19 @@ protected ResourceParameter keyFields() {
 
     @Override
     public AsType<ResourceParameter> sort() {
-        var sortString = parameters().get(SORT).as().toString();
-        if (missingIdentitySortKey(sortString)) {
-            parameters().set(SORT, sortString + COMMA + IDENTITY);
+        var sortString = parameters().get(SORT).toString();
+        if (missingIdentifier(sortString)) {
+            if (sortString.isBlank()) {
+                parameters().set(SORT, IDENTIFIER);
+            } else {
+                parameters().set(SORT, String.join(COMMA, sortString, IDENTIFIER));
+            }
         }
         return parameters().get(SORT);
     }
 
-    private static boolean missingIdentitySortKey(String sortString) {
-        return !sortString.contains(IDENTITY);
+    private static boolean missingIdentifier(String sortString) {
```

Comment:
nitpick: Since this is a boolean, perhaps rename with `has` or `is` prefix? I suspect it might be more readable without this helper method too, since it's so short.

Reply:
I tried with `hasMissingIdentifier`, but then the meaning changed....